engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/hv/MinAgeValidator.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.internal.constraintvalidators.hv;
+
+import org.hibernate.validator.constraints.MinAge;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+
+/**
+ *
+ * Checks that the number of years from a given date to today is greater or equal to
+ * a specified value
+ *
+ * @author Hillmer Chona
+ * @since 6.0.x
+ */
+
+
+public class MinAgeValidator implements ConstraintValidator<MinAge, LocalDate> {
```

Comment:
`LocalDateTime` seems worth supporting, too?

Reply:
Yeah, what I discussed with Hilmer is that we should have one type supported well. Then we would add the other types on the same model once we have converged.